Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM mozillamarketplace/centos-mysql-mkt:0.2
 
+# Copy the most recent git HEAD information so that we can debug
+# the state of the docker image better
+COPY .git/logs/HEAD /git-rev.txt
```

Comment:
for me this file has a ton of revisions. Could it be something more like `RUN git log --pretty=oneline -1 ...` ?